pkg/server/datastore/sqlstore/models.go

```diff
@@ -32,6 +32,7 @@ type AttestedNode struct {
 	ExpiresAt       time.Time `gorm:"index"`
 	NewSerialNumber string
 	NewExpiresAt    *time.Time
+	CanReattest     bool `gorm:"default:false"`
```

Comment:
Do we need this default? Since we aren't using a sql.NullBool or *bool, an unset column will be interpreted as `false` already... We don't set a default on our other bool fields (e.g. entry admin and downstream columns).

Reply:
You're right, I think we can safely rely on SQL defaults here